### PR TITLE
Prime Radiant Workspace models

### PR DESCRIPTION
**Pull request by simonrozsival**
This PR contains only definitions of the models to reduce the number of changed files in PR #398.

### CODE REVIEW

File: Toggl.PrimeRadiant/Models/IDatabaseWorkspaceFeature.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿using Toggl.Multivac.Models;
+
+namespace Toggl.PrimeRadiant.Models
+{
+    public interface IDatabaseWorkspaceFeature : IWorkspaceFeature, IDatabaseSyncable { }
```
Comment (simonrozsival) [inline Toggl.PrimeRadiant/Models/IDatabaseWorkspaceFeature.cs]: You are right, it is not needed, I will remove it.

Comment (simonrozsival) [inline Toggl.PrimeRadiant/Models/IDatabaseWorkspaceFeature.cs]: The only problem with that is that the generator expects every IDatabase.. to implement it, so the RealmConstructors must be edited to generate code which would compile.

Comment (amulware) [inline Toggl.PrimeRadiant/Models/IDatabaseWorkspaceFeature.cs]: Or we have to not generate the code for these types. I'm not sure what the best solution is right now.

**Comment by simonrozsival** [comment]
This PR contains just the models. The repository is in PR #398 - I split it in two to make it easier for reviewing.

The repository, which is included in `ITogglDatabase` exposes `IObservable<IDatabaseWorkspaceFeatureCollection> GetByWorkspaceId(long id)`. The `RealmWorkspaceFeatureCollection` exposes `bool IsEnabled(WorkspaceFeatureId feature)`.

**Comment by amulware** [comment]
Let's put this on hold until needed.

**Comment by simonrozsival** [comment]
This must be closed after #409 so the generator works correctly (now one line of the `FoundationConstructors.cs` file is manually edited).

**Comment by simonrozsival** [comment]
This is now ready for a review.

**Pull request merged**